error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -430,17 +430,19 @@ Flux<T> after(Flux<T> flux, T object) {
   }
 
   /** Prefer {@link Flux#empty()} over more contrived alternatives. */
-  // XXX: In combination with the `IsEmpty` matcher introduced by
-  // https://github.com/PicnicSupermarket/error-prone-support/pull/744, the non-varargs overloads of
-  // most methods referenced here can be rewritten as well. Additionally, some invocations of
-  // methods such as `Flux#fromIterable`, `Flux#fromArray` and `Flux#justOrEmpty` can also be
-  // rewritten.
+  // XXX: Using `@Matches(IsEmpty.class)`, the non-varargs overloads of most methods referenced here
+  // can be rewritten as well. That would require adding a bunch more suitably-typed parameters.
   static final class FluxEmpty<T, S extends Comparable<? super S>> {
+    // XXX: The methods enumerated here are not ordered entirely lexicographically, to accommodate a
+    // conflict between the `InconsistentOverloads` and `RefasterMethodParameterOrder` checks.
```

Comment:
This is something to be fixed, of course. But for now I just documented his, as I lost too much time trying to appease both.